src/StoragelessSession/Session/Data.php

```diff
@@ -39,6 +47,53 @@ private function __construct(array $data, array $metadata)
     {
         $this->data     = $data;
         $this->metadata = $metadata;
+        $this->expireDataFromScopes();
+    }
+
+    /**
+     * @param string $scopeName
+     *
+     * @return SessionScope
+     */
+    public function getScope(string $scopeName)
+    {
+        if (isset($this->scope[$scopeName])) {
+            return $this->scope[$scopeName];
+        }
+
+        return $this->scope[$scopeName] = new SessionScope(
+            $this->data[$scopeName] ?? [],
```

Comment:
We may need to check if `$this->data[$scopeName]` is an array (since it comes from mixed token data)